components/builder-protocol/protocols/depotsrv.proto

```diff
@@ -11,10 +11,11 @@ message Package {
   required PackageIdent ident = 1;
   required string checksum = 2;
   required string manifest = 3;
-  repeated PackageIdent deps = 4;
-  repeated PackageIdent tdeps = 5;
-  repeated uint32 exposes = 6 [packed=true];
-  optional string config = 7;
+  required string target = 4;
+  repeated PackageIdent deps = 5;
+  repeated PackageIdent tdeps = 6;
+  repeated uint32 exposes = 7 [packed=true];
+  optional string config = 8;
```

Comment:
ok per our discusion in standup, I moved target to the end along with its field number, recompiled and retested.